contracts/CO30_Matias_BlackSmokeRising.blud_cordelia/edit_me/description.ext

```diff
@@ -0,0 +1,11 @@
+// Mission settings - title, description, author etc.
+OnLoadName = "Black Smoke Rising";
+OnLoadMission = "Contractors clear camps and destroy weapon caches";
```

Comment:
Could be better if it was a bit more fleshed out but not entirely necessary. "Contractors are tasked with neutralizing enemy camps to prevent a possible military coup." or something along those lines.